torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -142,6 +145,31 @@ Node* insertQParam(
   return qparam;
 }
 
+Node* insertScalarToTensor(Graph* graph, Value* scalar_value) {
```

Comment:
These seem general purpose, should they be in one of the helper files?

Reply:
this is only used in insert_quant_dequant